src/test/resources/org/jenkinsci/plugins/casc/integrations/global-matrix-auth/GlobalMatrixStrategy.yml

```diff
@@ -0,0 +1,23 @@
+jenkins:
+  systemMessage: "Welcome!"
+  numExecutors: 4
+  scmCheckoutRetryCount: 2
+  mode: NORMAL
+  scmCheckoutRetryCount: 4
+  securityRealm:
+    local:
+      allowsSignup: false
+      users:
+       - id: test
+         password: test
+
+  authorizationStrategy:
+    globalMatrix:
+      grantedPermissions:
+        - name: "anonymous"
+          permissions:
+           - "hudson.model.Hudson.Read"
+        - name: "authenticated"
+          permissions:
+           - "hudson.model.Hudson.Administer"
```

Comment:
"hudson.model.Hudson" is a deprecated package. Use "jenkins.model.Jenkins" at least